docs/tutorials/testing_with_control_center.rst

```diff
@@ -38,11 +32,11 @@ Hardware Setup
   lead to dangerous voltage spikes when connected to a power supply. To prevent damage to the module when commanding it aggressively on a power supply, it is recommended to turn on the 
   regeneration voltage protection feature.
 
-To communicate with the motor using IQ Control Center, you will need:
+To communicate with the module using IQ Control Center, you will need:
 
 * A USB-to-UART converter (FTDI, CP2012, etc.)
-* Wires or a cable to connect your USB-to-UART converter to the motor's communication pins
-* A battery or power supply to connect to your motor
+* Wires or a cable to connect your USB-to-UART converter to the module's communication pins
+* A battery or power supply to connect to your module
 
 First, plug your USB-to-UART converter into your computer, and install any drivers if necessary 
```

Comment:
and install any necessary drivers (refer to the manufacturer's documentation for your...).